Testscripts/Windows/PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1

```diff
@@ -20,23 +20,28 @@ function Main {
         $noServer = $true
         $clientMachines = @()
         $clientIPs = ""
+        $numberOfReceivers = 0
+        $numberOfSenders = 0
         # role-0 vm is considered as the server-vm
         # role-1 vm is considered as the client-vm-01
```

Comment:
make this in one line as:
role-1 to role-8 are considered client-vm-01 to client-vm-08